default.json

```diff
@@ -83,12 +83,12 @@
     },
     {
       "id": 6,
-      "name": "jog in place",
+      "name": "high knees",
       "minReps": 20,
       "maxReps": 30,
```

Comment:
This is only a reference file, the actual config isn't in source control and needs to be updated by hand.